lethal-squirrels/workforce-management
# Navigation links

## REFERENCE

Ticket: #15

## DESCRIPTION
 - changed navigation links to show Index view of Employee, Departments, Computers, and Training Programs
- deleted unnecessary files (About.cshtml and Contact.cshtml)
## STEPS TO TEST
1. pull down the branch `mp-nav`
2. Test functionality
	* Run the program
	* click on the links in the nav bar
	* ensure the view displays the index view for each controller
